packages/api/src/es/search.js

```diff
@@ -96,7 +96,8 @@ const esConfig = async (host, metrics = false) => {
   return config;
 };
 
-class BaseSearch {
+// eslint-disable-next-line import/prefer-default-export, node/no-unsupported-features/es-syntax
+export class BaseSearch {
```

Comment:
Out of curiosity, why the need to ignore `node/no-unsupported-features/es-syntax`?

Reply:
The `ignore` is required because I added the `export`.

The `export` is required because, when `tsc` tried to build the `.d.ts` files for that file, it failed saying that `BaseSearch` was private. I don't entirely understand why, TS compiling JS seems a little magical. In any case, when that file and the files that use it are actually migrated over to `.ts`, that will go away.